loadtest/status.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package loadtest
+
+import (
+	"encoding/json"
+	"strings"
+	"time"
+)
+
+// State determines which state a loadtester is in.
+type State int
+
+// Different possible states of a loadtester.
+const (
+	StateUnknown State = iota
```

Comment:
Is there a reason we need this additional state? Isn't a newly created `LoadTester` stopped by default?

Reply:
This is required because of the json marshal/unmarshal thing. We need to map a state with some other unknown string.

Reply:
> This is required because of the json marshal/unmarshal thing. We need to map a state with some other unknown string.

I see. Can't we just fail if it's not one of those four? Why encode something that we know is not valid?